src/applications/disability-benefits/all-claims/content/summaryOfDisabilities.jsx

```diff
@@ -34,9 +33,6 @@ const mapDisabilityName = (disabilityName, formData, index) => {
 };
 
 const getRedirectLink = formData => {
-  if (environment.isProduction()) {
-    return 'go back to the beginning of this step and add it';
-  }
```

Comment:
This is an out of scope change, but I forgot this code was in there. We had to remove the `Link` due to some issues that have since been resolved.